<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeInfomationWidget</class>
 <widget class="QWidget" name="TimeInfomationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
     <item>
      <widget class="QLabel" name="BCodeTimeNameLabel">
       <property name="text">
        <string>B码时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="BCodeTimeLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <item>
      <widget class="QLabel" name="ntpTimeNameLabel">
       <property name="text">
        <string>NTP时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ntpTimeLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="systemTimeNameLabel">
       <property name="text">
        <string>系统时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="systemTimeLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>使用：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="BCodeRadioBtn">
       <property name="text">
        <string>B码时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ntpRadioBtn">
       <property name="text">
        <string>NTP时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="systemRadioBtn">
       <property name="text">
        <string>系统时间</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
